pkg/image/api/v1/types.go

```diff
@@ -308,7 +308,7 @@ type ImageStreamTag struct {
 	Conditions []TagEventCondition `json:"conditions,omitempty" protobuf:"bytes,4,rep,name=conditions"`
 
 	// Image associated with the ImageStream and tag.
-	Image Image `json:"image" protobuf:"bytes,5,opt,name=image"`
+	Image Image `json:"image" patchStrategy:"delete" protobuf:"bytes,5,opt,name=image"`
```

Comment:
There are [3 directives](https://github.com/soltysh/origin/blob/ed1d3328093830bd956089fab41be77ccb255fc1/vendor/k8s.io/kubernetes/pkg/util/strategicpatch/patch.go#L43-L45) from what I see. Although most of the time these are applied only to lists, including [delete](https://github.com/soltysh/origin/blob/ed1d3328093830bd956089fab41be77ccb255fc1/vendor/k8s.io/kubernetes/pkg/util/strategicpatch/patch.go#L539-L543). Besides, here `oc edit` ens up producing a patch that is later on submitted to server to modify istag.